index.html

```diff
@@ -11,22 +11,32 @@
 <body>
     <!--Header Starts Here-->
     <header class="header-container">
-        <div class="header-top">
+        <nav class="nav-top">
             <a href="#" class="header-logo">Welcome</a>
-            <button class="hamburger" type="button">
-                <i class="fa fa-solid fa-bars"></i>
+            <button class="hamburger" id="menu-bar" type="button">
+               <i class="fa fa-solid fa-bars"></i>
             </button>
-            <nav>
-                <ul class="header-nav-desktop">
+            <ul class="header-nav-desktop">
                     <li><a href="">Portifolio</a></li>
                     <li><a href="">About</a></li>
                     <li><a href="">Contact</a></li>
+            </ul>
+        </nav>
+        <div class="mobile-menu" id="menu-mobile">
+            <div class="menu-header">
+                <span class="close">&times;</span>
+            </div>
+            <div class="menu-body">
+                <ul>
+                    <li><a href="#work-id">Portfolio</a></li>
+                    <li><a href="#about-id">About</a></li>
+                    <li><a href="#contact-id">Contact</a></li>
```

Comment:
- [ ]  it would be better for the close button and links to add margin, padding in order to achieve the same design in the Figma 
- Yours 
<img width="377" alt="Screenshot 2022-11-14 at 15 19 29" src="https://user-images.githubusercontent.com/61790587/201683951-48c38f8f-d825-4e66-80a2-15e0e70a9563.png">
- Figma 
<img width="274" alt="Screenshot 2022-11-14 at 15 21 09" src="https://user-images.githubusercontent.com/61790587/201683982-d3d00803-9a71-4a65-ac6e-1838fce0e28e.png">